ads/_config.js

```diff
@@ -1106,6 +1106,11 @@ const adConfig = jsonConfiguration({
     ],
   },
 
+  'whopainfeed': {
+    prefetch: 'https://widget.infeed.com.ar/widget/widget-amp.js',
+    renderStartImplemented: true,
```

Comment:
I see you have this flag marked as true, but when I run your example ad I do not see this method being called. Can you confirm you are calling this API in your 3p code?